cmd/mistry/main.go

```diff
@@ -187,40 +212,24 @@ EXAMPLES:
 					}
 				}
 
-				jr := types.JobRequest{Project: project, Group: group, Params: params}
-				jrJSON, err := json.Marshal(jr)
-				if err != nil {
-					return err
+				url := fmt.Sprintf("http://%s:%s/%s", host, port, JobsPath)
+				if async {
+					url += "?async"
```

Comment:
Shouldn't we better use `async=true`? I think semantically it's more proper.

Reply:
not necessarily. URL params can support semantics similar to CLI boolean flags, i.e. the existence of the flag `--async` is what determines if it is true, absense of it determines it to be false. similarly in URL params, existence of the param `?async` or lack of it determines its value. unvalued URL params are supported by the spec

giving it an explicit value `?async=true` only makes sense if this is a three-valued boolean, i.e. if the absence of it also has meaning and we want to support the three values `true` `false` and `null`. we don't have this need here

Reply:
Makes sense!